### ISSUE TITLE
Feature request: DateIntervalFormatter

### ISSUE THREAD

**Issue by Strate** [jquery/globalize#620]
It weould be nice to have DateIntervalFormatter.
Implementation could be ported from https://github.com/papandreou/inter/blob/master/lib/inter.js#L825

**Comment by rxaviers** [comment]
Hi, I'm open for it.

I believe this is also called Date Range Format, another library that does that is iLib.
https://github.com/rxaviers/javascript-globalization/blob/master/date-range-format.md

Would you be interested in champion this? The first step I suggest is to look for alternatives and come up with an API.

**Comment by Strate** [comment]
I think I could do that, but only after #602 ;-)

**Comment by rxaviers** [comment]
;)
